packages/kbn-optimizer/src/worker/webpack.config.ts

```diff
@@ -162,7 +162,7 @@ export function getWebpackConfig(bundle: Bundle, worker: WorkerConfig) {
           ],
         },
         {
-          test: /\.scss$/,
+          test: /\.(scss|css)$/,
```

Comment:
We shouldn't be passing css to the sass-loader, should we? If the syntax should be parsed as sass I think we should change the extension to scss and not automatically parse it as scss.